WORKSPACE

```diff
@@ -30,7 +30,7 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_depe
 go_rules_dependencies()
 
 go_register_toolchains(
-    go_version = "1.11.5",
+    go_version = "1.12",
```

Comment:
`reflect.MapRange` is [used here](https://github.com/kubernetes/test-infra/pull/11622/files#diff-bb97e06c2a17f654b1eab01a05d73efeR286) and it's 1.12 method. I was hoping project version would be updated quite soon (as far as I know kubernetes/kubernetes is already on 1.12).
So I've added this to enable the feature and discuss if implementation looks ok. I would have to wait until project version is updated to avoid making this change.